runtime-modules/content/src/types.rs

```diff
@@ -87,19 +85,10 @@ pub struct ChannelRecord<MemberId: Ord, CuratorGroupId, AccountId, Balance> {
     pub moderators: BTreeSet<MemberId>,
     /// Cumulative cashout
     pub cumulative_payout_earned: Balance,
-}
-
-impl<MemberId: Ord, CuratorGroupId, AccountId, Balance>
-    ChannelRecord<MemberId, CuratorGroupId, AccountId, Balance>
-{
-    /// Ensure censorship status have been changed
-    pub fn ensure_censorship_status_changed<T: Trait>(&self, is_censored: bool) -> DispatchResult {
-        ensure!(
-            self.is_censored != is_censored,
-            Error::<T>::ChannelCensorshipStatusDidNotChange
-        );
-        Ok(())
-    }
+    // Privilege level (curators will have different moderation permissions w.r.t. this channel depending on this value)
+    pub privilege_level: ChannelPrivilegeLevel,
+    // Whether the channel has been marked as hidden by a curator
+    pub is_hidden: bool,
```

Comment:
We don't really need to store this, as it does not impact any transaction, the most recent _is_hidden_ just needs to be an even payload for a successful transaction which updates it, query node takes care of the rest. This means we cannot block a redundant state update where the state is set to the current value, but that is fine.

Reply:
Removed in https://github.com/Joystream/joystream/pull/3366/commits/3d6692e4f3234e6d2c35698ceb8a5619fb22e498